### PR TITLE
folks: new aport

### PR DESCRIPTION
**Pull request by Cogitri**

### CODE REVIEW

**Comment by Ikke** [comment]
Build error when building folks:

> backends/telepathy/lib/meson.build:19:0: ERROR: Static libraries can only be introspected with GObject-Introspection >=1.58.1

**Comment by Cogitri** [comment]
Ah, CI doesn't pull in the gobject-introspection bump :c

**Comment by Ikke** [comment]
Someone else has to do that, I don't have push access to main.

**Comment by Cogitri** [comment]
Ah, right.

**Comment by Ikke** [comment]
I built it locally, and there it indeed builds without issues. I did notice however that you comitted `gobject-introspection` after `folks`. It might be nice for other to commit them in dependency order so that it's easier to find out in what order to build changes.

**Comment by Cogitri** [comment]
Alright, will do :)

**Comment by Cogitri** [comment]
The g-i bump is in #7417 now.

**Comment by Cogitri** [comment]
Passes CI now.

**Comment by Ikke** [comment]
@fcolista Can you review the testing/evolution-data-server changes? Thanks

File: testing/telepathy-glib/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+pkgname=telepathy-glib
+pkgver=0.24.1
+pkgrel=0
+pkgdesc="Library for GLib-based Telepathy components"
+url="https://telepathy.freedesktop.org"
+arch="all"
+license="LGPL-2.1-or-later"
+makedepends="dbus-glib-dev intltool libxslt python glib-dev diffutils vala
+	gobject-introspection-dev"
+options="!check" # Glib-ERROR duplicate test case path => broken upstream
```
Comment (Ikke) [inline testing/telepathy-glib/APKBUILD]: This should be reported upstream it if it hasn't been already.

**Pull request closed without merging**